### PR TITLE
Adding Typescript declaration file for FocusManager API

### PR DESCRIPTION
**Pull request by samuelfreiberg**
Adding a Typescript declaration file for the FocusManager API. 

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-windows/pull/6834)

### CODE REVIEW

File: packages/react-native-win32/src/Libraries/Utilities/FocusManager.win32.d.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+ *
+ * @flow
+ * @format
+ */
+
+'use strict';
+
+import * as React from 'react';
+
+export declare class FocusManager {
```
Comment (NikoAri) [inline packages/react-native-win32/src/Libraries/Utilities/FocusManager.win32.d.ts:13]: >FocusManager [](start = 21, length = 12)

Why does this file have "win32" in it? It's already in react-native-win32 folder

**Review by NikoAri — APPROVED** [review]
:shipit:

File: packages/react-native-win32/src/Libraries/Utilities/FocusManager.win32.d.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * @flow
```
Comment (NickGerleman) [inline packages/react-native-win32/src/Libraries/Utilities/FocusManager.win32.d.ts:5]: We don't want Flow annotations in Typescript files

**Comment by NickGerleman** [comment]
Overall lgtm

File: packages/react-native-win32/src/Libraries/Utilities/FocusManager.win32.d.ts (hunk 2 above)
Comment (samuelfreiberg) [reply packages/react-native-win32/src/Libraries/Utilities/FocusManager.win32.d.ts]: Removed in newest commit

File: packages/react-native-win32/src/Libraries/Utilities/FocusManager.win32.d.ts (hunk 1 above)
Comment (samuelfreiberg) [reply packages/react-native-win32/src/Libraries/Utilities/FocusManager.win32.d.ts:12]: I was just following the patterns from the package. Every file in this Utilities folder has the ".win32"

Comment (NickGerleman) [reply packages/react-native-win32/src/Libraries/Utilities/FocusManager.win32.d.ts:12]: Platform extensions are related to how metro bundles platform specific files.

**Review by dannyvv — APPROVED** [review]

**Comment by dannyvv** [comment]
@samuelfreiberg: Feel free to merge at your convenience.

**Pull request merged**